install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -129,6 +255,21 @@ Brief description of each variable in the table below:
 |ansible_user| Ansible user used to deploy {product-title}. "openshift" is the required name and must not be changed.
 |openshift_openstack_disable_root| Boolean value that disables root access
 |openshift_openstack_user| OCP instances created with this user
+|openshift_openstack_node_network_name | Name of existing OpenShift network to use for deployment. This should be the same network name used for your deployment host.
+|openshift_openstack_node_subnet_name | Name of existing OpenShift subnet to use for deployment. This should be the same subnet name used for your deployment host.
+|openshift_openstack_router_name | Name of existing OpenShift router to use for deployment. This should be the same router name used for your deployment host.
+|openshift_openstack_master_floating_ip | Default is `true`. Must set to `false` if you do not want floating IPs assigned to master nodes.
+|openshift_openstack_infra_floating_ip | Default is `true`. Must set to `false` if you do not want floating IPs assigned to infrastructure nodes.
+|openshift_openstack_compute_floating_ip | Default is `true`. Must set to `false` if you do not want floating IPs assigned to compute nodes.
+|openshift_use_openshift_sdn | Must set to `false` if you want to disable openshift-sdn
+|openshift_use_kuryr | Must set to `true` if you want to enable kuryr sdn
+|use_trunk_ports | Must be set to `true` to create the OpenStack VMs with trunk ports (required by kuryr)
+|os_sdn_network_plugin_name | selection of the SDN behavior. Must set to `cni` for kuryr
+|openshift_node_proxy_mode | Must set to `userspace` for Kuryr
+|openshift_master_open_ports | Ports to be opened on the VMs when using Kuryr
+|kuryr_openstack_public_net_id | Need by Kuryr. ID of the public OpenStack network from where FIPs are obtained
+|openshift_kuryr_subnet_driver | Kuryr Subnet driver. Must be `namespace` for creating a subnet per namespace
```

Comment:
these 2 can be removed

Reply:
@luis5tb could you be specific which parameters? 
kuryr_openstack_public_net_id and openshift_kuryr_subnet_driver?

Reply:
sorry, I referred to openshift_kuryr_subnet_driver and openshifT_kuryr_sg_driver